apps/montreal_elixir/lib/montreal_elixir/social_feeds/social_feeds.ex

```diff
@@ -0,0 +1,26 @@
+defmodule MontrealElixir.SocialFeeds do
+  @moduledoc """
+  The SocialFeeds context.
+  """
+
+  alias MontrealElixir.SocialFeeds.Cache
+  alias MontrealElixir.SocialFeeds.MeetupApiClient
+
+  @doc """
+  Returns the next (upcoming) meetup_event.
+  If there is no upcoming event, returns the last (recent) meetup_event.
+
+  This function does not hit external API if there is an unexpired next_meetup_event in cache.
+
+  ## Examples
+
+      iex> get_next_meetup_event()
+      %MeetupEvent{}
+
+  """
+  def get_next_meetup_event do
+    Cache.fetch(:next_meetup_event,
+                fn -> MeetupApiClient.get_next_event() end,
+                %{expires_in: 600})
```

Comment:
It was not obvious to me that this was in seconds, since we usually deal with milliseconds in most of the OTP modules.  Maybe add a comment?  Of maybe we should have a configuration item such as `cache_ttl_in_sec`?  Not sure though.

Reply:
Sorry about that (still living 95% in Ruby world), I can change it to milliseconds.